src/routes/post.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const router = express.Router();
+
+const postController = require('../controllers/post');
+
+router.get('/', postController.getAllPosts);
+router.post('/add', postController.addPost);
+router.get('/filter', postController.filterPosts);
+router.get('/:id', postController.getOnePost);
+router.put('/:id', postController.updatePost);
+router.delete('/:id', postController.removePost);
+router.post('/like/:id', postController.updateLikes);
```

Comment:
Change this to `/:id/likes`.
Then the full route becomes `/api/posts/:id/likes`.
You can read it right to left as "likes of post id of posts".